### PR TITLE
Some more fixes for the chat page

### PR DESCRIPTION
**Pull request by dngray**
https://deploy-preview-2012--privacytools-io.netlify.app/services/chat/

### CODE REVIEW

File: pages/services/chat.html [hunk 1]
```diff
@@ -50,7 +50,7 @@ <h4>Is #general:privacytools.io or #privacy:privacytools.io chat end-to-end encr
 <p><strong>No.</strong> Messages sent to the #general chat are encrypted from with client-to-server encryption, i.e. HTTPS. This means that your messages can't be viewed by an attacker on your network, or your ISP, etc. <strong>However</strong>, because this is a public chat room anybody can view your messages if they are a member, and newcomers will be able to see all message history. Do not say anything in rooms that don't use End-to-End (E2E) encryption that you wouldn't want tied to you personally.</p>
 
 <h4>Isn't Matrix behind Cloudflare?</h4>
-<p>Matrix.org is, chat.privacytools.io isn't. If you use our homeserver and our Element install, you should be good.</p>
+<p>The default matrix.org homeserver is behind Cloudflare. The PrivacyTools homeserver (<a href="https://element.privacytools.io/">element.privacytools.io</a> is not.
```
Comment (Mikaela) [inline pages/services/chat.html:53]:
```suggestion
<p>The default matrix.org homeserver is behind Cloudflare. The PrivacyTools homeserver (<a href="https://chat.privacytools.io/">chat.privacytools.io</a> is not.
```

Element still seems to be the webclient while chat. says to be the homeserver.

Comment (dngray) [reply pages/services/chat.html:53]: Ah yes. good point.

**Comment by FreddyMarsden** [comment]
Related: [#11](https://github.com/privacytools/static-sites/pull/11)

**Comment by dngray** [comment]
We should probably look at https://github.com/privacytools/privacytools.io/issues/2015 as well for this issue.

File: pages/services/chat.html [hunk 2]
```diff
@@ -50,15 +50,15 @@ <h4>Is #general:privacytools.io or #privacy:privacytools.io chat end-to-end encr
 <p><strong>No.</strong> Messages sent to the #general chat are encrypted from with client-to-server encryption, i.e. HTTPS. This means that your messages can't be viewed by an attacker on your network, or your ISP, etc. <strong>However</strong>, because this is a public chat room anybody can view your messages if they are a member, and newcomers will be able to see all message history. Do not say anything in rooms that don't use End-to-End (E2E) encryption that you wouldn't want tied to you personally.</p>
 
 <h4>Isn't Matrix behind Cloudflare?</h4>
-<p>Matrix.org is, chat.privacytools.io isn't. If you use our homeserver and our Element install, you should be good.</p>
+<p>The default matrix.org homeserver is behind Cloudflare. The PrivacyTools homeserver (<a href="https://chat.privacytools.io/">chat.privacytools.io</a> is not.
```
Comment (jonaharagon) [inline pages/services/chat.html:53]:
```suggestion
<p>The default matrix.org homeserver is behind Cloudflare. The PrivacyTools homeserver is not.
```

File: pages/services/chat.html [hunk 3]
```diff
@@ -19,7 +19,7 @@ <h1 class="display-4">Matrix</h1>
 </div>
 
 <h2 class="pt-4" id="homeserverinfo">Connection Information</h2>
-<p><strong>To register on or connect to our homeserver, simply use <code>https://chat.privacytools.io</code> as the homeserver in your Matrix client.</strong></p>
+<p><strong>To register on or connect to our homeserver, simply use <code>https://chat.privacytools.io</code> as the homeserver in your Matrix client. <p><strong>Please see our community <a href="https://wiki.privacytools.io/view/PrivacyTools:Code_of_Conduct">code of conduct</a> before joining our rooms or using our services.</strong></p>
```
Comment (jonaharagon) [inline pages/services/chat.html:22]:
```suggestion
<p><strong>To register on or connect to our homeserver, simply use <code>https://chat.privacytools.io</code> as the homeserver in your Matrix client.</strong></p>
<p><mark>You must read and accept our community <a href="https://wiki.privacytools.io/view/PrivacyTools:Code_of_Conduct">code of conduct</a> before joining our rooms or using our services.</mark></p>
```

Fix crazy formatting

**Comment by dngray** [comment]
It seems this didn't take place on the live page https://www.privacytools.io/services/chat @jonaharagon.

Two things I noticed that didn't seem to make it was the CoC, and the refinement of the Cloudflare related question.

**Pull request merged**